webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -1183,24 +1186,35 @@ func (v validator) validateMinHardwareVersion(ctx *pkgctx.WebhookRequestContext,
 	var allErrs field.ErrorList
 	fieldPath := field.NewPath("spec", "minHardwareVersion")
 
-	// Disallow downgrades.
-	oldHV, newHV := oldVM.Spec.MinHardwareVersion, vm.Spec.MinHardwareVersion
-	if newHV < oldHV {
+	// kubebuilder validations handles check for minHardwareVersion < 13.
+	// This is necessary here to handle brownfield api versions.
+	if vimtypes.HardwareVersion(vm.Spec.MinHardwareVersion) > vimtypes.MaxValidHardwareVersion {
 		allErrs = append(allErrs, field.Invalid(
 			fieldPath,
 			vm.Spec.MinHardwareVersion,
-			invalidMinHardwareVersionDowngrade))
+			fmt.Sprintf(invalidMinHardwareVersionNotSupported, vimtypes.MaxValidHardwareVersion)))
 	}
 
-	// Disallow upgrades unless powered off or will be powered off.
-	if newHV > oldHV {
-		if oldVM.Spec.PowerState != vmopv1.VirtualMachinePowerStateOff &&
-			vm.Spec.PowerState != vmopv1.VirtualMachinePowerStateOff {
-
+	if oldVM != nil {
+		// Disallow downgrades.
+		oldHV, newHV := oldVM.Spec.MinHardwareVersion, vm.Spec.MinHardwareVersion
+		if newHV < oldHV {
 			allErrs = append(allErrs, field.Invalid(
 				fieldPath,
 				vm.Spec.MinHardwareVersion,
-				invalidMinHardwareVersionPowerState))
+				invalidMinHardwareVersionDowngrade))
+		}
+
+		// Disallow upgrades unless powered off or will be powered off.
```

Comment:
thanks. Added a release note. let me know if that looks okay. 

> I'm curious why this? Just because it was missing before? Or was this not needed previously and is now?
Not a new change. I've only moved the existing upgrade/downgrade validations into a block where the oldVM is not nil.